templates/hadoop/conf/hdfs-site.xml

```diff
@@ -9,6 +9,6 @@
 
   <property>
     <name>dfs.datanode.data.dir</name>
-    <value>/mnt/hdfs</value>
+    <value>{root_ephemeral_dirs}</value>
```

Comment:
`{root_ephemeral_dirs}` will map to `/media/ephemeral0/hdfs,/media/ephemeral1/hdfs,...` or `/media/root/hdfs`, depending on whether the instance offers ephemeral storage.

Flintrock does that by [passing in the module](https://github.com/nchammas/flintrock/pull/59/files#diff-1ec778e3c6cde832d96a7c08ba31f98eR120) that the template is being formatted for, so each module gets its own space. So Spark's config files, for example, get `.../spark` for the same `{root_ephemeral_dirs}` template variable.

Does that answer your question?